CHANGELOG.md

```diff
@@ -4,7 +4,15 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
-## [Unreleased]
+## [Unreleased
```

Comment:
Meh?

Was this updated with `FUTURE_RELEASE=2.3.0 bundle exec rake changelog`?

Reply:
Now, fixed.

Reply:
> Was this updated with `FUTURE_RELEASE=2.3.0 bundle exec rake changelog`?
No, where is the documentation about this?

Reply:
` ## [Unreleased]` should not appear since it's a release :stuck_out_tongue_closed_eyes:.  That ChangeLog was managed by hand, but we now have all the tooling to manage it based on the PR merged ;-)  Yes we loose some early changes, but I think this really improves the situation

Reply:
Fine